core/src/app/halma/AboutScreen.java

```diff
@@ -65,6 +73,12 @@ public void clicked(Actor a) {
             link = "https://github.com/mmelnizky";
         if(a == linkToMarvin)
             link = "https://github.com/Crazy-Marvin";
+        if (a == helpWIthTranslation)
+            link = "https://hosted.weblate.org/engage/halma/";
+        if (a == termsOfService)
+            link = "https://github.com/Crazy-Marvin/Halma/blob/development/LICENSE";
+        if (a == privacyPolicy)
+            link = "https://github.com/Crazy-Marvin/Halma/blob/trunk/.github/CONTACT.md";
```

Comment:
This is the link to the Privacy Policy: https://app.getterms.io/view/7FsFk/privacy/en-us